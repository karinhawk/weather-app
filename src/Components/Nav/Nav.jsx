import "./Nav.scss"

const Nav = (toggleView) => {
  return (
    <div className="nav">
        <button onClick={toggleView}>Local Weather</button>
        <button onClick={toggleView}>Choose Location</button>
    </div>
  )
}

export default Nav