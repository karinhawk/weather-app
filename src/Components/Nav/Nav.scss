.nav {
    height: 10vh;
    border-bottom: 1px solid rgb(212, 211, 211);

    &__bar {
        height: 10vh;
        display: flex;
        justify-content: space-between;
        align-items: center;
    }

    &__options {
        height: 10vh;
        width: 85vw;
        display: flex;
        justify-content: space-around;
        align-items: center;
    }
    
   &__menu {
    margin-left: 1rem;
   }
    
}