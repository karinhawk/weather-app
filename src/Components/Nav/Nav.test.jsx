import Nav from "./Nav.jsx"
import { render } from "@testing-library/react";

it("Should render the nav", () => {
    const weatherData = {
        location: "London"
    }
    const {asFragment} = render(<Nav weatherData={weatherData}/>)
    expect(asFragment(<Nav />)).toMatchSnapshot()
  });