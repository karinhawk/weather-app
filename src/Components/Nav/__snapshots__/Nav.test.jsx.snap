// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the nav 1`] = `
<DocumentFragment>
  <div
    class="nav"
  >
    <div
      class="nav__bar"
    >
      <img
        class="nav__menu"
        src="menu.png"
      />
      <div
        class="nav__options"
      >
        <div>
          <button
            class="button"
          >
            Local Weather
          </button>
        </div>
        <form
          class="form"
        >
          <input
            class="form__searchbox"
            placeholder="Town... City... Country..."
            type="text"
          />
          <button
            class="form__button"
            type="submit"
          >
            Go
          </button>
        </form>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
